modules/ztp-image-based-upgrade-creating-backup-resources-ztp.adoc

```diff
@@ -0,0 +1,170 @@
+// Module included in the following assemblies:
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-creating-backup-resources-with-ztp_{context}"]
+= Creating ConfigMap objects to back up resources with GitOps ZTP
+
+The {lcao} provides functionality for backing up and restoring platform and application resources with the OADP Operator.
+
+The {lcao} automatically extracts and applies site-specific manifests that have the `ran.openshift.io/ztp-deploy-wave` annotation and the `lca.openshift.io/target-ocp-version: "<target_version>"` label.
+These manifests are extracted and applied after rebooting to the new version during the `Upgrade` stage.
+
+.Prerequisites
+
+* Install the OADP Operator in the `source-crs` directory for the target cluster.
+* Prepare the `DataProtectionApplication` CR and its secret in the `source-crs` directory for the target cluster.
+* Create an S3-compatible storage solution and a ready-to-use bucket with proper credentials configured. For more information, see _Additional resources_.
+
+.Procedure
+
+[NOTE]
+====
+The target cluster does not need to be detached from the hub cluster.
```

Comment:
IIUC we have this note because previously we needed to detach the target cluster.
The thing is that now this note is more confusing (i.e. why mentioning the detach).  If we don't mention the detaching anywhere else - we can just remove it.